doc/man3/ASN1_INTEGER_new.pod

```diff
@@ -0,0 +1,44 @@
+=pod
+
+=head1 NAME
+
+ASN1_INTEGER_new, ASN1_INTEGER_free, ASN1_INTEGER_it -
+ASN1_INTEGER allocation functions
+
+=head1 SYNOPSIS
+
+=for openssl generic
+
+ #include <openssl/asn1.h>
+
+ extern const ASN1_INTEGER_it;
+ ASN1_INTEGER *ASN1_INTEGER_new(void);
+ void ASN1_INTEGER_free(ASN1_INTEGER *a);
```

Comment:
I do wonder whether we would be better having a single "TYPE" page that defines these functions for all the similar ASN1 functions, i.e. something similar to the d2i_X509.pod page, but for new/free etc.

https://github.com/openssl/openssl/blob/master/doc/man3/d2i_X509.pod

Reply:
Well, much better!

Reply:
i must drop this page and combit it with others ? Or I must fix this PR

Reply:
Fixing this PR will be better, I think.

Reply:
> i must drop this page and combit it with others ? Or I must fix this PR

Fixing this PR to add a new "generic" page for all ASN1 *_new/*_free functions would be the ideal solution.

Note - I'm not absolutely requiring you to do this as part of this PR if you don't want to. It's a suggestion for how we could make this much better - but its a fair amount of work to do - and what you have here is better than what we have at the moment (i.e. nothing). So if you don't want to make a generic page - then fine. I will review this PR for what it is and maybe someone else can make that change later.